### PR TITLE
[SCM-925] Implement RemoveCommand in maven-scm-provider-jgit

### PR DESCRIPTION
**Pull request by G-Ork**

### CODE REVIEW

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.apache.maven.scm.command.remove.RemoveScmResult;
+import org.apache.maven.scm.provider.ScmProviderRepository;
+import org.apache.maven.scm.provider.git.command.GitCommand;
+import org.apache.maven.scm.provider.git.jgit.command.JGitUtils;
+import org.eclipse.jgit.api.Git;
+
+/**
+ * @author <a href="mailto:tsakumagos@gmail.com">Georg Tsakumagos</a>
+ * @author Georg Tsakumagos
+ * @since 1.11.2.1
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java]: This is obviously wrong.

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+public class JGitRemoveCommand extends AbstractRemoveCommand implements GitCommand {
+
+	
+	@Override
+	protected ScmResult executeRemoveCommand(ScmProviderRepository repository, ScmFileSet fileSet, String message)
+			throws ScmException {
+
+        if ( fileSet.getFileList().isEmpty() )
+        {
+            throw new ScmException( "You must provide at least one file/directory to remove (e.g. -Dincludes=...)" );
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java]: Make the message consistent with the `GitExeProvider`.

**Comment by michael-o** [comment]
Can you also please come up with a test case?

File: maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java (hunk 1 above)
Comment (G-Ork) [reply maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/remove/JGitRemoveCommand.java]: Could you spell out the obviously wrong?

**Comment by G-Ork** [comment]
It's been a while. Fix issues hopefully with another PR.

**Pull request closed without merging**